source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -39,18 +39,22 @@ open class AdaptiveCard {
 }
 
 public struct RenderConfig {
-    public static let `default` = RenderConfig(isDarkMode: false, buttonConfig: .default, supportsSchemeV1_3: false, hyperlinkColorConfig: .default)
+    public static let `default` = RenderConfig(isDarkMode: false, buttonConfig: .default, supportsSchemeV1_3: false, hyperlinkColorConfig: .default, checkButtonConfig: .default, radioButtonConfig: .default)
     let isDarkMode: Bool
     let buttonConfig: ButtonConfig
     // swiftlint:disable identifier_name
     let supportsSchemeV1_3: Bool
     let hyperlinkColorConfig: HyperlinkColorConfig
+    let checkButtonConfig: ChoiceSetButtonConfig
+    let radioButtonConfig: ChoiceSetButtonConfig
```

Comment:
```suggestion
    let checkButtonConfig: ChoiceSetButtonConfig?
    let radioButtonConfig: ChoiceSetButtonConfig?
```
This should be optional. If it's nil, then system radio button and checkbox should be shown